src/snapshot/src/crc.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
What year do we typically put in there? Is it the first time this was written, or the last time it was touched?

Reply:
Answering my own question: this should be the year that we first published this code. This was taken from the versionize crate: https://github.com/firecracker-microvm/versionize/blob/main/src/crc.rs, so I think that I should just leave 2020. WDYT?

Reply:
Ah that's fine by me... I think I have also seen 2020-2024, but it would be the only one in the repo to have a year range.

Reply:
Ok, I stayed with the 2020 year. Another issue going the year range is that it would imply that we need to update the range in all the files once per year.